Specs/Core/loadKTX2Spec.js

```diff
@@ -1,7 +1,7 @@
 import { loadKTX2 } from "../../Source/Cesium.js";
+import { KTX2Transcoder } from "../../Source/Cesium.js";
```

Comment:
I think there should be a `KTX2TranscoderSpec` testing the parsing code, while `loadKTX2Spec` focuses on error/undefined handling?